JSONLD/auxillary/mip_era/cmip6.json

```diff
@@ -0,0 +1,8 @@
+{
+    "@type": "cmip:mip_era",
+    "@id": "mip-cmor-tables:auxillary/mip_era/cmip6",
+    "mip:name": "CMIP6",
+    "mip:url": "https://wcrp-cmip.org/CMIP6",
+    "mip:start": "2999",
```

Comment:
Ditto here, planning started 2013, first data July 2018, publication continuing today

Reply:
is there a planning close?

Reply:
Planning and reality are not the same thing, but yes, I would assume that after 2025 there will be zero additional publications to the project, we might even manage to get this done before the end of 2024, but we'll see - requires the ESGF nodes and SOLR index management involvement to do this